specification/redis/Cache.Management/RedisCacheAccessPolicyAssignment.tsp

```diff
@@ -0,0 +1,70 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.Cache;
+/**
+ * Response to an operation on access policy assignment
+ */
+@parentResource(RedisResource)
+model RedisCacheAccessPolicyAssignment
+  is Azure.ResourceManager.ProxyResource<RedisCacheAccessPolicyAssignmentProperties> {
+  ...ResourceNameParameter<
+    Resource = RedisCacheAccessPolicyAssignment,
+    KeyName = "accessPolicyAssignmentName",
+    SegmentName = "accessPolicyAssignments",
+    NamePattern = "^([a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9]|[a-zA-Z0-9])$"
+  >;
+}
+
+@armResourceOperations
+interface RedisCacheAccessPolicyAssignments {
+  /**
+   * Gets the list of assignments for an access policy of a redis cache
+   */
+  get is ArmResourceRead<RedisCacheAccessPolicyAssignment>;
+
+  /**
+   * Adds the access policy assignment to the specified users
+   */
+  createUpdate is ArmResourceCreateOrReplaceAsync<
+    RedisCacheAccessPolicyAssignment,
+    LroHeaders = ArmLroLocationHeader & Azure.Core.Foundations.RetryAfterHeader
```

Comment:
Missing final result.

Reply:
Isn't final result implicit from ArmLroLocationHeader?
I.e. you should get the resource again.

Reply:
> Isn't final result implicit from ArmLroLocationHeader?

If the Response is customized, then the LroHeaders is replaced, which contains the final result information.

Reply:
@pshao25 I'm feeling slightly confused about this one. Do we need to customize the response here? What is the custom bit.